docs/Using-Virtual-Environment.md

```diff
@@ -0,0 +1,33 @@
+# Using Virtual Environment
+
+## What is a Virtual Environment?
+A Virtual Environment is a walled garden for Python projects.  
```

Comment:
Would move the first line of the next section to "what is ..".  For example:

A Virtual Environment is a walled garden for Python projects.  This allow you to keep all dependencies for the project separate from dependencies of other projects.

And remove the first line in the 2nd section

Reply:
"walled garden" sounds pretty strange here (and might be confusing for non-native speakers). Can we just quote from (or link to) the official docs, e.g. https://docs.python.org/3/tutorial/venv.html 
> a self-contained directory tree that contains a Python installation for a particular version of Python, plus a number of additional packages.

Reply:
Done